src/scenes/home/informationForm/formComponents/militaryInfo.js

```diff
@@ -1,29 +1,48 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { Line } from 'rc-progress';
 import Form from 'shared/components/form/form';
 import PropTypes from 'prop-types';
 import FormSelect from 'shared/components/form/formSelect/formSelect';
-import { MILSTATUS, BRANCH } from 'shared/constants/status';
+import { MILSTATUS, BRANCH, BRANCH_PROMPT } from 'shared/constants/status';
 import styles from './formComponents.css';
 
-const MilitaryInfo = ({ percent, update }) => (
-  <Form className={styles.signup}>
-    <h3>Progress = {percent}%</h3>
-    <Line percent={percent} strokeWidth="4" strokeColor="green" />
-    <FormSelect
-      id="militaryStatus"
-      options={MILSTATUS}
-      prompt="Current Military Status"
-      onChange={e => update(e, e.target.value)}
-    />
-    <FormSelect
-      id="branch"
-      options={BRANCH}
-      prompt="Your Branch of Service"
-      onChange={e => update(e, e.target.value)}
-    />
-  </Form>
-);
+class MilitaryInfo extends Component {
+  constructor(props) {
+    super(props);
+    this.onChange = this.onChange.bind(this);
```

Comment:
Since you've used arrow syntax for `onChange` this line is actually unnecessary! [I've loved getting rid of binding statements](https://medium.freecodecamp.org/react-binding-patterns-5-approaches-for-handling-this-92c651b5af56) 🍾.